viewmaster/viewmaster.py

```
@@ -1993,11 +2273,11 @@ def viewmaster(query_path):
                 LOGGER.info('Returning fancy index', url)
                 return redirect(url + '?viewmaster_referrer=' + http_referrer)
             except Exception:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)